iohub/ngff.py

```diff
@@ -1564,6 +1566,61 @@ def positions(self) -> Generator[tuple[str, Position], None, None]:
         for _, well in self.wells():
             for _, position in well.positions():
                 yield position.zgroup.path, position
+    
+    def rename_wells(self, well_names_csv: str):
```

Comment:
I think this signature should be `rename_well(self, old: str, new: str):`. See Ziwen's `rename_channel` for an example of what we have in mind here.